server/app/api.py

```diff
@@ -1821,7 +1763,8 @@ def get_courses(self, course, user, data):
     def get_students(self, course, user, data):
         query = models.Participant.query(
             models.Participant.course == course.key,
-            models.Participant.role == 'student')
+            models.Participant.role == 'student',
+            models.Participant.status != 'inactive')
```

Comment:
@alvinwan These and the above are not changes I have made - they are probably from merges. When I merge with master, it says that my branch is already up-to-date though. Are the two places you pointed out the only two that should be changed? If so, I can do it manually.